test/e2e/expectedfailures/large-workflow.yaml

```diff
@@ -3,15 +3,1572 @@ kind: Workflow
 metadata:
   generateName: large-workflow-
 spec:
-  entrypoint: large-workflow
```

Comment:
@jessesuen added this file in #913 but it doesn't look like the e2e tests use it.